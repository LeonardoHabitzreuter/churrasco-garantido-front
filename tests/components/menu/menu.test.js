import React from 'react'
import { shallow } from 'enzyme'
import Menu from '../../../src/components/menu'

describe('Menu component', () => {
  test('should renders properly', () => {
    const wrapper = shallow(<Menu />)
    expect(wrapper).toMatchSnapshot()
  })
})
