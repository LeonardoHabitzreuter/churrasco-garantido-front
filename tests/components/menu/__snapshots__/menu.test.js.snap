// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component should renders properly 1`] = `
<Menu
  className=""
  defaultSelectedKeys={
    Array [
      "blank",
    ]
  }
  mode="inline"
  onClick={[Function]}
  prefixCls="ant-menu"
  theme="dark"
>
  <MenuItem
    key="/"
  >
    <Icon
      type="dashboard"
    />
    <span>
      Dashboard
    </span>
  </MenuItem>
  <MenuItem
    key="/registerCompany"
  >
    <Icon
      type="form"
    />
    <span>
      Cadastrar empresa
    </span>
  </MenuItem>
  <SubMenu
    key="orders"
    title={
      <span>
        <Icon
          type="shopping-cart"
        />
        <span>
          Pedidos
        </span>
      </span>
    }
  >
    <MenuItem
      key="/orders/newOrder"
    >
      <Icon
        type="plus"
      />
      <span>
        Novo pedido
      </span>
    </MenuItem>
    <MenuItem
      key="/orders/myOrders"
    >
      <Icon
        type="table"
      />
      <span>
        Meus pedidos
      </span>
    </MenuItem>
  </SubMenu>
  <MenuItem
    key="/myAccount"
  >
    <Icon
      type="user"
    />
    <span>
      Minha conta
    </span>
  </MenuItem>
  <MenuItem
    key="/logout"
  >
    <Icon
      type="logout"
    />
    <span>
      Sair
    </span>
  </MenuItem>
</Menu>
`;
